src/settings/_color.scss

```diff
@@ -1,36 +1,52 @@
-// --- Colors --- //
+///////////////////////////////////////////////////////////////////////////////
+// Color Palette
+///////////////////////////////////////////////////////////////////////////////
 
-// Main
-$c-primary-light: #80a9ca !default;
+// System color
+$c-system: #000 !default;
+
+// Primary color
 $c-primary: #005496 !default;
-$c-primary-dark: #003661 !default;
-$c-highlighted: #f2cd96 !default;
+$c-primary-light: light-variation($c-primary) !default;
+$c-primary-dark: dark-variation($c-primary) !default;
 
+// Secondary/Accent color
 $c-accent: #964200 !default;
-$c-accent-light: #caa080 !default;
-$c-accent-dark: #612a00 !default;
+$c-accent-light: light-variation($c-accent) !default;
+$c-accent-dark: dark-variation($c-accent) !default;
 
-$c-featured: #ffe613 !default;
+// Gray color
+$c-gray: #777777 !default;
+$c-gray-light: light-variation($c-gray) !default;
```

Comment:
These variable values ar incorrect.
Variation color values are unified by UX team, so we cannot use custom values.
We should use the [Color-Variation Function](https://github.com/SUI-Components/theme-basic/blob/master/src/utils/_colors.scss#L15) instead to avoid having a plethora of random variations.

Reply:
well UX also defines the step of darker en lighter color which is 2. So I think it's ok to have the corresponding func.